### DISCUSSION TITLE
R2C在后面发布的几个版本,都无法使用LAN口,WAN口可用

### DISCUSSION THREAD

**Discussion by icrazy65536** [community/community#56736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

作者你好,我在R2C上测试过,今年发布的版本,都无法使用LAN口,WAN口可以使用(LAN口直接不亮,看不到任何数据,我知道接口有互换),我甚至下载了R2S等其他型号的压缩包,但是测试结果一样.
不知道新版是否能修复这个问题,感谢!

**Comment by queenofcorgis** [comment]
Hi @icrazy65536 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
